src/provider/model/AppConnection.ts

```diff
@@ -8,63 +8,46 @@ import {IntentType} from '../intents';
 import {Application, ChannelId} from '../../client/main';
 
 import {ContextChannel} from './ContextChannel';
+import {EntityType} from './Environment';
 
 /**
- * Model interface, representing a window that has connected to the service.
+ * Model interface, representing an entity that has connected to the service.
  *
- * Only windows that have created intent or context listeners will be represented in this model. If any non-registered
- * window.
- *
- * TODO [SERVICE-737] Review naming of this interface, due to these objects now representing (potentially window-less)
- * external connections. Likewise for "downstream" types/functions/etc.
+ * Since an IAB connection is required for an `AppConnection` object to be created, these will only ever refer to FDC3-enabled windows/applications/etc.
```

Comment:
Would use `entity` instead of `windows/applications/etc.`, since I'd say this definitely isn't an application